src/wasm-lib/kcl/src/ast/types.rs

```diff
@@ -2257,13 +2258,46 @@ pub enum BinaryOperator {
     #[serde(rename = "%")]
     #[display("%")]
     Mod,
+    /// Raise a number to a power.
+    #[serde(rename = "^")]
```

Comment:
`Pow` being `^` does make sense from a math point of view (specifically those coming from LaTeX, etc), but Python, Rust, C, Go, etc use `**` for exponentiation. We may want to document this pretty heavily and/or come out with a "We don't do bitwise maths" which may help avoid confusion since `XOR` wouldn't ever be present.

Reply:
Yeah, I think given our audience (mech eng) and that we never manipulate network etc, we're probably not going to support bitwise operations.